models/settings_helpers.go

```diff
@@ -44,6 +45,8 @@ func GetSettings(q reform.DBTX) (*Settings, error) {
 	}
 
 	s.fillDefaults()
+	s.DBaaS.Enabled = os.Getenv("PERCONA_TEST_DBAAS") == "1"
```

Comment:
Nope, we already set this value in `envvars` package, just use it in `supervisord`